### ISSUE TITLE
Image Wrapping & Wrap plugin

### ISSUE THREAD

**Issue by shawndibble** [splitbrain/dokuwiki-plugin-dw2pdf#175]
1) I have images that are set to float left or right and have the text wrap around them. This works find for when I print as well, however when I send my book to pdf, the text does not wrap around them even though the images are aligned to the right/left. Is there a way to correct this issue?

![no-wrap](https://cloud.githubusercontent.com/assets/1980293/6399639/61806952-bdb9-11e4-8880-757bbecf1117.PNG)

2)So I use the wrap plugin. I have tried adding multiple pages from my wiki, each with some notification/info/alert via the wrap plugin or I just do an entire namespace. The first wiki page generates just fine with the wrap style present. However after that, the CSS drops.

Here is the correct and pdf output as you can see in the PDF, this wiki page is lacking the wrap styling. But like I said, the first embedded wiki page will retain the wrap styling.

![print](https://cloud.githubusercontent.com/assets/1980293/6399474/455e2210-bdb8-11e4-950f-d23c2ee8c06f.PNG)
![pdf](https://cloud.githubusercontent.com/assets/1980293/6399475/45611cb8-bdb8-11e4-8f48-c620e5af06b8.PNG)

**Comment by LarsGit223** [comment]
Hi,

I searched for the reason for point 2) but couldn't find one. After quickly replacing the mpdf folder with the newest version mpdf6.0 the problem was gone :sweat_smile:.

But I had to comment out 2 source code lines which mpdf complained about because they are deprecated. I will go deeper into that in the next days and if I can sort it out I will provide a pull-request including the new mpdf version and adjustments for the deprecated lines.

Cheers,
Lars
